spec/features/course/homepage_spec.rb

```diff
@@ -6,10 +6,23 @@
 
   with_tenant(:instance) do
     let(:course) { create(:course, :opened) }
-    let(:achievement_feed_notification) do
-      achievement = create(:course_achievement)
+    let(:feed_notifications) do
```

Comment:
Okay, discussed this, for now we can leave it as it is. 

In the future if we do find ourselves repeating the building of certain objects for these notification factories, we can think about whether we can move some of it into the factory file.